server/views/partials/variationDetails.njk

```diff
@@ -1,48 +1,36 @@
 {% from "customComponents/summaryCard/macro.njk" import summaryCard %}
 
```

Comment:
Updated this macro to display the conversation as well as SPO / VLO answers. May need to split these into two distinct summary cards to match the design more closely.